.github/workflows/industrial_ci_action.yaml

```diff
@@ -12,10 +12,21 @@ jobs:
       CCACHE_DIR: /github/home/.ccache
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v4.1.1
+      - uses: actions/cache@v3.3.2
```

Comment:
Just curious here; you've pinned newer versions here. Was there a particular reason for that (like a specific feature), or was v2 just out of date?

Reply:
Received warnings about those versions are out of date and wouldn't be supported in the near future. So, @dave992 asked me to proceed with the latest versions.